README.md

```diff
@@ -159,7 +159,8 @@ installation has the support or not for algorithms 15 and 16, respectively.
 
 All supported OSs, except CentOS Linux 7, support algorithms 15 and 16 out of the
 box. To get the support in CentOS Linux 7 a newer version of OpenSSL has to be
-installed and Zonemaster-LDNS has to be instlled following special instructions.
+installed and Zonemaster-LDNS has to be installed following special instructions
+found in the [Zonemaster-Engine] installation instructions.
```

Comment:
Did you consider updating the previous paragraph as well? It refers to a table that does not exist anymore.

Maybe something like:

> To be able to support and process algorithms 15 (Ed25519) and 16 (Ed448) for DNSSEC the underlying OS must have a recent version of [OpenSSL] installed, and [LDNS] being linked against that OpenSSL (see [Zonemaster-LDNS-README][Zonemaster-LDNS] for more details). A test of the domains `ed25519.nl` and `superdns.nl` will reveal if the Zonemaster installation has the support or not for algorithms 15 and 16, respectively.

Reply:
I did not see your comment, but I did some updates. Please review.